src/components/Pagination.js

```diff
@@ -44,7 +44,10 @@ const Pagination = (props) => {
             }
           }}
         >
-          <EOS_KEYBOARD_ARROW_LEFT className='eos-icons eos-18' />
+          <EOS_KEYBOARD_ARROW_LEFT
+            color={`${currNumber <= 1 ? '#bbb' : ''}`}
```

Comment:
I don't like this solution. It is only changing the icon color, but the element will remain in the DOM. We need to skip the element injection in the DOM altogether